speech/cloud-client/README.md

```diff
@@ -14,6 +14,12 @@ using the [Google Cloud Client Library for Java][google-cloud-java].
 
 ## Setup
 
+Before you begin, complete the following steps as listed on the [Speech-to-Text Docs pages for Getting Started](https://cloud.google.com/speech-to-text/docs/quickstart-client-libraries#before-you-begin)
+ * Create or select a project.
```

Comment:
This line might want to specifically call out `GOOGLE_APPLICATION_CREDENTIALS`

Reply:
Added a line below